index.html

```diff
@@ -4,9 +4,320 @@
     <meta charset="utf-8">
     <title>HTML Academy: Глейси</title>
   </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+  <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <img class="logo" src="" alt="Логотип Gllacy Shop">
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="catalog.html">Каталог</a>
+            <ul class="navigation-list navigation-list-catalog">
+              <li class="navigation-item">
+                <a class="navigation-link navigation-link-new" href="new.html">Новинки</a>
+              </li>
+              <li class="navigation-item">
+                <a class="navigation-link" href="creamy.html">Сливочное</a>
+              </li>
+              <li class="navigation-item">
+                <a class="navigation-link" href="sherbets.html">Щербеты</a>
+              </li>
+              <li class="navigation-item">
+                <a class="navigation-link" href="fruit_ice.html">Фруктовый лед</a>
+              </li>
+              <li class="navigation-item">
+                <a class="navigation-link" href="melorin.html">Мелорин</a>
+              </li>
+            </ul>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="delivery_and_payment.html">Доставка и оплата</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="about.html">О компании</a>
+          </li>
+        </ul>
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item ">
+            <a class="navigation-link navigation-telephone" href="#">+7 800 555-86-28</a>
```

Comment:
Ну вот у телефона точно можно прописать ссылку, есть даже отдельный формат записи:
```suggestion
            <a class="navigation-link navigation-telephone" href="tel:+78005558628">+7 800 555-86-28</a>
```

Reply:
поправила